packages/fxa-content-server/app/scripts/lib/fxa-client.js

```diff
@@ -111,11 +111,19 @@ function getUpdatedSessionData(email, relier, accountData, options = {}) {
     uid: accountData.uid,
     verificationMethod: accountData.verificationMethod,
     verificationReason: accountData.verificationReason,
-    verified: accountData.verified || false,
-    metricsEnabled: accountData.metricsEnabled || false,
+    verified: accountData.verified,
+    metricsEnabled: accountData.metricsEnabled,
```

Comment:
Q: Do we want to use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here (versus a separate check+block below on L119-125)?
Something like:

```js
    verified: accountData.verified ?? false,
    // Note, `metricsEnabled` will always default to true.
    metricsEnabled: accountData.metricsEnabled ?? true,
```

Reply:
This is nicer. Thanks for the rec. I will update it.